src/main/kotlin/org/jitsi/nlj/rtcp/RembHandler.kt

```diff
@@ -42,7 +48,21 @@ class RembHandler(parentLogger: Logger) : RtcpListener {
     }
 
     private fun onRembPacket(rembPacket: RtcpFbRembPacket) {
+        if (streamInformationStore.supportsTcc) {
+            logger.warn { "Ignoring unexpected REMB, when using TCC (for ${rembPacket.bitrate.bps} bps)" }
```

Comment:
Don't we want the warn message to be guarded by sawSpuriousRemb so we don't spam the logs?

Reply:
I mean, to have the log once ... :)